examples/AN00221_freq_domain_example/src/mic_array_conf.h

```diff
@@ -2,9 +2,18 @@
 #ifndef MIC_ARRAY_CONF_H_
 #define MIC_ARRAY_CONF_H_
 
+#define MIC_ARRAY_WORD_LENGTH_SHORT 1
 
-#define MIC_ARRAY_MAX_FRAME_SIZE_LOG2 9
+#if MIC_ARRAY_WORD_LENGTH_SHORT
+//#define MIC_ARRAY_MAX_FRAME_SIZE_LOG2 12 // 4k FFT. Builds but there is no audio.
+#define MIC_ARRAY_MAX_FRAME_SIZE_LOG2 9 // 512 FFT
```

Comment:
I think setting it to 9 and including explanation in the app not would be cleaner